src/topology/continuous_function/algebra.lean

```diff
@@ -305,6 +306,38 @@ coe_injective.comm_group _ coe_one coe_mul coe_inv coe_div coe_pow coe_zpow
     exactI λ K hK, uniform_continuous_inv.comp_tendsto_uniformly_on
       (tendsto_iff_forall_compact_tendsto_uniformly_on.mp filter.tendsto_id K hK), } }
 
+-- TODO: rewrite the next three lemmas for products and deduce sum case via `to_additive`, once
+-- definition of `tprod` is in place
+
+/-- If `α` is locally compact, and an infinite sum of functions in `C(α, β)`
+converges to `g` (for the compact-open topology), then the pointwise sum converges to `g x` for
+all `x ∈ α`. -/
+lemma has_sum_apply {α : Type*} {β : Type*} {γ : Type*}
+  [add_comm_monoid β] [topological_space α] [locally_compact_space α] [topological_space β]
+  [has_continuous_add β]
+  {f : γ → C(α, β)} {g : C(α, β)} (hf : has_sum f g) (x : α) :
+  has_sum (λ i : γ, f i x) (g x) :=
+begin
+  let ex : add_monoid_hom C(α, β) β :=
+  { to_fun := (λ f, f x),
+    map_zero' := by simp only [coe_zero, pi.zero_apply],
+    map_add' := by simp only [coe_add, pi.add_apply, eq_self_iff_true, forall_const] },
```

Comment:
I think you can build this more concisely by composition with `coe_fn_monoid_hom` and another hom that exists elsewhere

Reply:
Would you mind elaborating on that?

Reply:
`continuous_map.coe_fn_monoid_hom` has the same domain as this function, so you should be able to compose it with `library_search` to find the other half.